activerecord/lib/active_record/railtie.rb

```diff
@@ -236,6 +228,18 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_record.set_config", after: :load_config_initializers do |app|
```

Comment:
This also wrok for `config/initializers` but Railties will still not be able to set config. I think this should happen as late is possible, maybe in an `after_initializer`. That would avoid the need to use the `after:` option.

Reply:
Yeah, I tried that before and it wouldn't work, but it now does. Quite puzzling. Either way i think it's good to go now.